src/main/java/net/imagej/ops/descriptor3d/QuickHull3D.java

```diff
@@ -0,0 +1,545 @@
+package net.imagej.ops.descriptor3d;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Set;
+
+import org.apache.commons.math3.geometry.euclidean.threed.Vector3D;
+import org.scijava.plugin.Plugin;
+
+import net.imagej.ops.AbstractFunctionOp;
+import net.imagej.ops.Op;
+import net.imagej.ops.Ops.Descriptor3D.ConvexHull3D;
+
+/**
+ * This quickhull implementation is based on the paper
+ * "The Quickhull Algorithm for Convex Hulls" by Barber, Dobkin and Huhdanpaa
+ * (http://dpd.cs.princeton.edu/Papers/BarberDobkinHuhdanpaa.pdf).
+ * 
+ * The computation of the initial simplex is inspired by John Lloyd's quickhull
+ * implementation (http://www.cs.ubc.ca/~lloyd/java/quickhull3d.html).
+ * 
+ * @author Tim-Oliver Buchholz, University of Konstanz
+ *
+ */
+@Plugin(type = Op.class, name = ConvexHull3D.NAME)
+public class QuickHull3D extends AbstractFunctionOp<HashSet<Vertex>, DefaultFacets> implements ConvexHull3D {
```

Comment:
Never type things on `DefaultFoo`. They should be typed on `Foo` interface that backs it. The `Default` class should have _no new public API_ in it—but only implement the methods in the interface.